src/mlpack/tests/convolutional_network_test.cpp

```diff
@@ -47,11 +47,13 @@ BOOST_AUTO_TEST_CASE(VanillaNetworkTest)
   {
     if (i < nPoints / 2)
     {
-      Y(i) = 4;
+      // class 1 - digit = 4
```

Comment:
Pedantic style issue  it's helpful to always have comments that are complete sentences with proper grammar and punctuation.